csharp/src/Google.Protobuf.Test/Collections/RepeatedFieldTest.cs

```diff
@@ -599,5 +600,61 @@ public void IList_Insert()
             list.Insert(1, "middle");
             CollectionAssert.AreEqual(new[] { "first", "middle", "second" }, list);
         }
+
+        [Test]
+        public void ToString_Integers()
+        {
+            var list = new RepeatedField<int> { 5, 10, 20 };
+            var text = list.ToString();
+            Assert.AreEqual("[ 5, 10, 20 ]", text);
```

Comment:
Do other languages also generate the spaces inside [ ] and { } - ["a", "b"] vs [ "a", "b" ]?

From https://developers.google.com/protocol-buffers/docs/proto3?hl=en#json  is looks like they might not.